Source/WebCore/dom/Document.cpp

```diff
@@ -8080,6 +8082,57 @@ bool Document::resolveViewTransitionRule()
     return false;
 }
 
+void Document::reveal()
```

Comment:
Add a link to https://html.spec.whatwg.org/multipage/browsing-the-web.html#revealing-the-document